app/views/ThreadMessagesView/index.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
```

Comment:
![WhatsApp Image 2020-10-28 at 1 26 02 PM](https://user-images.githubusercontent.com/29778115/97465944-57a55400-1921-11eb-8249-964a72028cfa.jpeg)
:top: What about this weird behavior?

Reply:
Use something like this: https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2506/commits/3002c24cb907dfdf7c0026713bc1bd7f2590fe9e

Reply:
Nice workaround!